<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPlainTextEdit" name="recvEdit">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>501</width>
     <height>321</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>340</y>
     <width>711</width>
     <height>166</height>
    </rect>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>96</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileNameEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="infoLab">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>94</width>
      <height>98</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="sendBtn">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendStopbtn">
       <property name="text">
        <string>停止发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openFileBtn">
       <property name="text">
        <string>打开文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>294</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="sendNewLineChx">
       <property name="text">
        <string>发送新行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="timerSendChx">
       <property name="text">
        <string>定时发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>周期:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="timerPeriodEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="sendFileBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendclearBtn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清除发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>181</width>
     <height>261</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>14</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>串口选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baudRateCmb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="stopBitsCmb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="dataBitsCmb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="parityCmb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>校验位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openPortBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearRecvBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清除接收</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portsCmb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="closePortBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="freshBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="hexDisplayChx">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>100</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>16进制接收</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sendHexChx">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>16进制发送</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
